distrobox-create

```diff
@@ -166,9 +166,20 @@ generate_command() {
 
 # First, check if the image exists in the host
 if ! podman image exists "${container_image}"; then
-	echo "Image not found, run this command first:"
-	echo "	podman pull ${container_image}"
-	exit 1
+    echo >&2 "Image not found."
+    echo >&2 "Do you want to pull the image now?[y/n]"
```

Comment:
By default if I do not enter anything and just go with `enter` it defaults to Yes, so I'd put the y capital in [Y/n]

Reply:
More than that, I'd not default to Yes but to No, just because this potentially could cause a big download.